HW7.py

```diff
@@ -0,0 +1,21 @@
+"""
+Homework 7. If-elif-else.
+
+Given values w,x,y,z
+write if-elif-else statement that will search minimum value
+and print smth aka "'y' is minimum value'
+where 'y' is variable name (not value)
+advice use python debugger and different values to check your algorithm
+optionally you can check your algorithm somehow with assert statements
+"""
+
+w, x, y, z = 100, 0, -40, 30
```

Comment:
Close...  but algorithm should be improved. Now it has gaps. 
If to use such data
w, x, y, z = -1, -1, -1, 0
result will be 'z' , but should be "w" or "x" or "y"